### DISCUSSION TITLE
quicksearch, search with and without special chars

### DISCUSSION THREAD

**Discussion by teafmartins** [serenity-is/Serenity#5326] [Help]
Hello,
Any way in the quick search to include searching for special characters? 
Example, if I search for "Jérôme" I should find records with "Jérôme" and "Jerome".

Thanks and Regards

**Comment by edwardch** [comment]
The query that hits your database on the quicksearch will add a `WHERE` filter like so: `WHERE T0.[Name] LIKE '%Jerome%'`.

Depending on your database's collation you might get different results. You can specify collation in your queries directly on the database (see link below), but I have not tried this in Serenity yet. Search for some collation flags, or something similar. Hope this link helps your understanding a bit...

https://stackoverflow.com/questions/2461522/how-do-i-perform-an-accent-insensitive-compare-e-with-%C3%A8-%C3%A9-%C3%AA-and-%C3%AB-in-sql-ser
